R/ComputeFirstDifferences.R

```diff
@@ -0,0 +1,53 @@
+#' Adds first differences of a variable, robustly to missing values, as new columns in a panel dataset
+#'
+#' @param df Data frame that will be modified
+#' @param idvar Character indicating column of grouping variable.
```

Comment:
```suggestion
#' @param idvar Character indicating column of units.
```

For parallelism with `EventStudy.R`:
https://github.com/JMSLab/eventstudyr/blob/27d5f4ff836ce809c8cf5376db4db1dedc30f96a/R/EventStudy.R#L9